package.json

```diff
@@ -31,7 +31,7 @@
     "copy-files": "cpy src/index.html src/favicon.ico dist",
     "docker:run": "webpack-dev-server --inline --progress --port 8080",
     "functests": "npm-run-all -p webdriver-start testserver-start protractor",
-    "lint": "tslint ./src/*",
+    "lint": "tslint -c tslint.json --force src/**/*.ts src/**/**/*.ts src/**/**/**/*.ts src/**/**/**/**/*.ts src/**/**/**/**/**/*.ts",
```

Comment:
I see what are you trying to do here, but I believe just **src/**/**.ts** should do the trick.